build.sbt

```diff
@@ -1,82 +1,52 @@
-javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint")
-
-lazy val commonSettings = Seq(
-  organization := "Dwolla",
-  homepage := Option(url("https://stash.dwolla.net/projects/OPS/repos/cloudflare-public-hostname-lambda")),
+lazy val buildSettings = Seq(
+  organization := "com.dwolla",
+  name := "cloudformation",
+  homepage := Some(url("https://github.com/Dwolla/scala-cloudflare")),
+  description := "Scala library for the Cloudflare v4 API",
+  licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
+  releaseVersionBump := sbtrelease.Version.Bump.Minor,
+  releaseProcess := {
```

Comment:
This release process excludes the publish step. I think since we're not building for multiple Scala versions we can remove this customization and rely on the default `releaseProcess`.